mne/stats/cluster_level.py

```diff
@@ -455,6 +454,8 @@ def _find_clusters(x, threshold, tail=0, connectivity=None, max_step=1,
                 else:
                     len_c = len(c)
                 scores[c] += h * (len_c ** e_power)
+    # turn sums into ndarray after running
+    sums = np.concatenate(sums) if sums else np.empty(0)
```

Comment:
`sums` is 1D so it's simpler just to do `sums = np.array(sums)` as `np.array([])` is the same as `np.empty(0)`

Reply:
`sums` is a list of `np.array`, if we do `sums = np.array(sums)` it breaks other functions.
I would change it to `sums = np.concatenate(sums) if sums else np.array([])`.